magento/merchdocs#60: Update product review information (#1276)
magento/merchdocs: src/marketing/magento-product-reviews-moderate.md

To ensure that reviews are appropriate for public display your store, a submitted Magento product review must be approved before it can be displayed. A submitted review is in a `Pending` status until it is approved or rejected.

1. On the _Admin_ sidebar, go to **Marketing** > _User Content_ > **Pending Reviews** or **All Reviews**.

1. In the list, click a pending review to view the details and edit if necessary.

1. Change the **Status** according to your assessment:

   - To approve a pending review, select `Approved`.

   - To reject a review, select `Not Approved`. Unapproved reviews disappear from the list of **Pending Reviews** page.

> 1. Also, the admin can moderate the **Detailed Rating**, **Nickname**, **Summary of Review** fields.

Reviewer: ```suggestion
1. If needed, change the values for **Detailed Rating**, **Nickname**, and **Summary of Review** .
```
